tools/integration_tests/interrupt/interrupt_test.go

```diff
@@ -28,13 +33,26 @@ const (
 	testDirName = "InterruptTest"
 )
 
+var (
+	storageClient *storage.Client
```

Comment:
I don't think we need this global variable

Reply:
Done